.travis.yml

```diff
@@ -0,0 +1,9 @@
+language: python
+python:
+  - "3.4"
+  - "3.5"
+  - "3.6"
+install:
+  - pip install django djangorestframework
```

Comment:
this may need a "sudo" im not 100% sure

Reply:
Not necessarily, @tapegun. Check out [this](https://docs.travis-ci.com/user/getting-started/#Selecting-infrastructure-(optional))